whylog/tests/tests_config/__init__.py

```diff
@@ -16,46 +16,51 @@
 # convertions
 to_date = "date"
 
-regex1 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Connection error occurred on (.*)\. Host name: (.*)$"
-regex2 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data migration from (.*) to (.*) failed\. Host name: (.*)$"
-regex3 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data is missing at (.*)\. Loss = (.*) GB\. Host name: (.*)$"
-
-parser_intent1 = UserParserIntent("connectionerror", "hydra", regex1, [1], {1: to_date})
-parser_intent2 = UserParserIntent("datamigration", "hydra", regex2, [1], {1: to_date})
-parser_intent3 = UserParserIntent("lostdata", "filesystem", regex3, [1], {1: to_date})
+path_test_files = ['whylog', 'tests', 'tests_config', 'test_files']
 
-parsers = {0: parser_intent1, 1: parser_intent2, 2: parser_intent3}
-effect_id = 2
 
-constraint1 = UserConstraintIntent(identical_constr, [[0, 2], [1, 2]])
-constraint2 = UserConstraintIntent(identical_constr, [[1, 3], [2, 2]])
-constraint3 = UserConstraintIntent(different_constr, [[1, 2], [1, 3]])
-constraint4 = UserConstraintIntent(hetero_constr, [[0, 3], [1, 4], [2, 4]], {"different": 1})
+class TestBasic(TestCase):
+    def setUp(self):
```

Comment:
This is done again and again for every test in this class. Since those objects are immutable, they can be not recreated for every test